cmd/run_konan

```diff
@@ -89,12 +89,13 @@ SHARED_JAR="${KONAN_HOME}/konan/lib/shared.jar"
 EXTRACTED_METADATA_JAR="${KONAN_HOME}/konan/lib/konan.metadata.jar"
 EXTRACTED_SERIALIZER_JAR="${KONAN_HOME}/konan/lib/konan.serializer.jar"
 KLIB_JAR="${KONAN_HOME}/konan/lib/klib.jar"
+KLIOPT_JAR="${KONAN_HOME}/konan/lib/kliopt-jvm.jar"
```

Comment:
Not sure if I like this approach. Can we avoid adding new JARs and repack to fewer JARs?

Reply:
We discussed with Ilya, we need to think about right repacking all jars, but for now it's only possible approach without big refactoring.